cms/search/utils.py

```diff
@@ -16,12 +16,14 @@ def build_standard_resource_dict(page: "Page") -> dict:
     """Returns a dict with the standard resource fields.
     This covers the non-release case (and also forms the base of the release case).
     """
+    title = page.display_title if page.search_index_content_type == "statistical_article" else page.title
```

Comment:
Nit, use `isinstance` instead of checking `search_index_content_type` if we want this for only the stats article, or `getattr(page, "display_title", page.title)` if it should apply to any page with a display title.

This also needs a test as it is a fix.

Reply:
Was going to use `isinstance`, then my thought went to we only have `display_title` property for `StatisticalArticlePage` model far as I know. 

Going with `getattr(page, "display_title", page.title)` makes sense to me.

Failing tests fixed and updated the factory instantiation to `StatisticalArticlePageFactory(news_headline="")` have been updated to reflect this change also. This ensures our article pages are tested to make sure they're getting the title from the `display_title` property.

[758f915](https://github.com/ONSdigital/dis-wagtail/pull/349/commits/758f915f9ef24e9d564913ad82ca95fe841d50f9)